.github/workflows/mautic-asset-upload/composer.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "mautic/upload-release-asset",
+    "description": "Updates the release asset (x.x.x.zip) to the m.mautic.org instance",
+    "type": "project",
+    "require": {
+        "mautic/api-library": "^2.16"
```

Comment:
You can probably guess why I had to use the `2.16` version here and not `3.0`... 🤐